### ISSUE TITLE
Updating ban message

### ISSUE THREAD

**Issue by Jonathan-5** [TeamUlysses/ulx#45]
### Steps to reproduce
1. Ban a minge for prop spamming and trying to crash your server
2. Go to bans
3. Add the reason because you rushed to ban them

### Expected behavior
Error will pop up the first time in console.

### Actual behavior
Lag for a sec but works...

### Error(s) in player's console, if any
[ERROR] addons/ulx/lua/ulx/xgui/bans.lua:200: attempt to index local 'bandata' (a nil value)
  1. ShowBanDetailsWindow - addons/ulx/lua/ulx/xgui/bans.lua:200
   2. DoDoubleClick - addons/ulx/lua/ulx/xgui/bans.lua:14
    3. OnClickLine - lua/vgui/dlistview.lua:478
     4. unknown - lua/vgui/dlistview_line.lua:93

### Version
ULib v2.60
ULX v3.70

**Comment by Nayruden** [comment]
To confirm, you ban the user without a reason, then go to update the reason from XGUI?

**Comment by Jonathan-5** [comment]
Found it!
When you update a ban and you're done you click "Update" and then you hit the "Close" button but the tricky thing was and I just found how the error happens, RIGHT AFTER you hit close and click on another item IN THE LIST OF BANS the ERROR will SHOW, every time...

**Comment by Jonathan-5** [comment]
Actually it looks like the error pops up when you click the same ban in the list, not another one

**Comment by Jonathan-5** [comment]
```
[ERROR] addons/ulx/lua/ulx/xgui/bans.lua:200: attempt to index local 'bandata' (a nil value)
  1. ShowBanDetailsWindow - addons/ulx/lua/ulx/xgui/bans.lua:200
   2. DoDoubleClick - addons/ulx/lua/ulx/xgui/bans.lua:14
    3. OnClickLine - lua/vgui/dlistview.lua:478
     4. unknown - lua/vgui/dlistview_line.lua:93


```

**Comment by Jonathan-5** [comment]
While you hit "update" and "close" the ban list starts to refresh but as soon as you click on the same ban you edited, before its done refreshing the error will happen.

**Comment by SticklyMan** [comment]
Fixed in 57136874314287c30d83301f4618802ac62d1fd1
